self-registration/app/templates/index.html

```diff
@@ -0,0 +1,42 @@
+{% extends "base.html" %} {% block content %}
+<div class="card content-center">
+  <div class="card-header">
+    <h1>Account Registration</h1>
+  </div>
+  <div class="card-body">
+    <p>
+      Submit your email address and a valid coupon code to receive a trial
+      account for our AI platform.
```

Comment:
Hi @jbouder thanks for bringing the Nebari theme to this!! I wonder if we could have a `card_message` var or similar to modify the card-body paragraph more easily. 
```suggestion
      {% if card_message %} {{ (card_message| safe) }} {% else %} Submit your email address and a valid coupon code to receive a trial account for our AI platform. {% endif %}
```

Reply:
Great idea, i'll get that in the PR.

Reply:
Updated as requested